src/cart/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth import get_user_model
+from django.core.validators import MinValueValidator
+from django.db import models
+
+
+class Cart(models.Model):
+    user = models.ForeignKey(to=get_user_model(), on_delete=models.CASCADE)
+    creation_date = models.DateTimeField(auto_now_add=True)
+    payment_type = models.CharField(max_length=100, null=True)
+    payment_id = models.CharField(max_length=100, null=True)
+    ordered = models.BooleanField(default=False)
+
+
+class CartItem(models.Model):
+    part = models.ForeignKey("catalogue.Part", on_delete=models.CASCADE)
+    quantity = models.IntegerField(
```

Comment:
positive

Reply:
Ok, I will change

Reply:
After change, we can delete the validator, doesn't it?